src/harness/fake_sas.py

```diff
@@ -192,6 +192,9 @@ def GetEscSensorRecord(self, request, ssl_cert=None, ssl_key=None):
       # Return Empty if invalid Id
       return {}
 
+  def GetFullActivityDump(self):
```

Comment:
Thanks for the changes! I think the function signature is not matching with an interface `def GetFullActivityDump(self, ssl_cert=None, ssl_key=None):`

Also, you need to add this to `do_GET()` in fake_sas.py?